pkg/component/plutono/plutono.go

```diff
@@ -489,6 +490,11 @@ func (p *plutono) getDashboards(ctx context.Context) (map[string]string, error)
 }
 
 func (p *plutono) getService() *corev1.Service {
+	servicePort := port
+	if p.values.IsGardenCluster {
+		servicePort = 80
+	}
```

Comment:
Why do we need this? Can't we use the  same port in both scenarios i.e. remove the `if` statement?